addOns/ascanrules/CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+## [41] - 2021-08-12
+### Added
+- The XSS scan rule will try several different payloads if the payload is being
+reflected outside of any HTML tags (for example in a json response body).
+
```

Comment:
This should be left in the Unreleased section, the version and date are updated as part of the release process. Also it's a change (maybe a fix) not "Added".

```suggestion
### Changed
- The XSS scan rule will try several different payloads if the payload is being reflected outside of any HTML tags (for example in a json response body).

```